src/logger.rs

```diff
@@ -39,6 +39,76 @@ impl Default for LoggerConfig {
     }
 }
 
+pub struct Logger {
```

Comment:
I think ci is upset because there are no docs, but an example here would be really helpful.

Reply:
Yeah I will write the documentation when everything else is ok